active/Resources/StyleVersion2.md

```diff
@@ -0,0 +1,89 @@
+ThemeVersion API
+=
+
+# Background
+
+XAML (WinUI) has a resources feature where you can put objects in a "resource dictionary", then
+use the resources by referencing the key name. All of the styling for XAML controls
+is defined in a resource dictionary and referenced by the controls.
+But an application can override these by defining a resource dictionary with one or more matching keys.
+
+XAML ships such an overrides dictionary named
+[XamlControlsResources](http://msdn.microsoft.com/library/Microsoft.UI.Xaml.Controls.XamlControlsResources).
+This dictionary defines all the resources necessary for all of the built-in controls.
+Since the resources in this dictionary can change over time, a new `ThemeVersion` property is being added.
+
+> StylesVersion? ResourcesVersion?
+
+The default value of this property will be the latest version, but an app can set another value to force to a
+previous version.
+
+# API Pages
+
+## XamlControlsResources.ThemeVersion
+
+Set this property to specify which version of resources to use for the look and styling
+of XAML controls. Default value is Version2.
+
+Version 1 aligns to WinUI 2.5, Version 2 aligns with WinUI 2.6.
+
+> This property must be set during app startup and cannot be changed after that point.  
+(When, exactly?)
+
+> Spec note: the alternative is to have an 'Auto' value in the enum,
+and then the default value of the property is always that.
+That would mean, though, that we also need an ActualThemeVersion property for
+controls to query

```

Comment:
Or we do it this way: Each time the theme is refreshed, we introduce a new enum for it, which becomes the new default.

So you always get what the property says, and you can peek at the default value to see what the system recommends.

Reply:
That's the plan in this spec. The spec note here is calling out the alternative that was rejected.